nservicebus/tools/migrate-to-native-delivery.md

```diff
@@ -209,19 +209,17 @@ Use `SELECT * FROM TimeoutsMigration_State` to list all performed migrations. Fo
 
 ### NHibernate persistence
 
- - Delete all the entries from `StagedTimeoutEntity`, that contains the copy of migrated timeouts
- - Delete the table from `MigrationsEntity` that contains a row for every performed migration
+ - Delete all rows from the `StagedTimeoutEntity` table. These are copies of migrated timeouts.
+ - Delete the `MigrationsEntity` table. Each row in this table represents a previously performed migration.
```

Comment:
Can the entire table be deleted or just the rows?

Reply:
The entire table may be deleted once the migration is complete. It's a one-time thing, we have it in place only for troubleshooting or verification purposes.

Reply:
Sorry, I put this comment on the wrong line. I was referring to the `StagedTimeoutEntity` table. Can that table be deleted or just the rows?